### ISSUE TITLE
where is results.json

### ISSUE THREAD

**Issue by lorvent** [Suberbia/UltimateChatRestorer#1]
Hello,
I exported a chat history with a person but i don't find any results.json file

and without that file, i think this importer doesn't work, can you please tell me any workaround?

thanks.

**Comment by goshante** [comment]
i think he means result.json file which is produced by telegram chat export on PC when you choose to export as JSON instead of HTML

**Comment by lorvent** [comment]
yes, thanks..... i tried that and it is working well :)

only thing is my export is in html and i should find a way to convert that to json.

thanks

> i think he means result.json file which is produced by telegram chat export on PC when you choose to export as JSON instead of HTML
> 
> btw, this script is not actually what are you looking for, it restores only text messaes, ignoring all content. You better try this: https://github.com/filippz/telegram_import It's really working and imports everything

**Comment by uensal** [comment]
> yes, thanks..... i tried that and it is working well :)
> 
> only thing is my export is in html and i should find a way to convert that to json.
> 
> thanks
> 
> > i think he means result.json file which is produced by telegram chat export on PC when you choose to export as JSON instead of HTML
> > btw, this script is not actually what are you looking for, it restores only text messaes, ignoring all content. You better try this: https://github.com/filippz/telegram_import It's really working and imports everything

I habe the same problem
@lorvent  did you find a way?

**Comment by goshante** [comment]
>only thing is my export is in html and i should find a way to convert that to json.

> I habe the same problem

Are you guys blind?
![image](https://github.com/Suberbia/UltimateChatRestorer/assets/37217019/acbf6a5f-f0e0-4468-9c49-0ba2a0547785)

**Comment by uensal** [comment]
@goshante this doe snot help at all, if you exported it as html, then deleted everything.. then everything is gone and there is nothing any more to export the data as json.. i tried to export it afterwards, but there is nothing to be exported, since its gone... obviously :-(. So what is left is a backup file that was once exported as html in the past

**Comment by goshante** [comment]
@uensal You definitely doing something wrong. Check everything, increase limits, pick JSON, export anb boom, you have it.
It worked for me multiple times. Just make a new folder, choose it and export into it as JSON.

**Comment by uensal** [comment]
@goshante Nope, unfortunately not working.. after clearing the history for both everything is gone. No matter what i try, i only get the messages exported that i truly have there.

**Comment by goshante** [comment]
@uensal You had to tell about it right from the beginning. There is no way to restore removed conversation. Once it's removed it is lost forever.
If you managed somehow to save it as HTML before removing you have to make a script or something like that to format it into JSON that is acceptable for above mentioned script.

**Comment by lorvent** [comment]
> > yes, thanks..... i tried that and it is working well :)
> > only thing is my export is in html and i should find a way to convert that to json.
> > thanks
> > > i think he means result.json file which is produced by telegram chat export on PC when you choose to export as JSON instead of HTML
> > > btw, this script is not actually what are you looking for, it restores only text messaes, ignoring all content. You better try this: https://github.com/filippz/telegram_import It's really working and imports everything
> 
> I made another code that would work on .html exports: https://github.com/Suberbia/html_to_txt

wow, thank you so much for this.

my concern is....does it import all images, gifs, stickers etc too (as the json version does) ?

**Comment by lilium895** [comment]
Hi, I've made a repository for restoring telegram html back up. Starting from html_to_txt of Suberbia.  
It works and it restore also the medias.  
The method is slow, though.
If you want it to check it up and maybe change the code for your purpose it's there.

**Comment by BestChessPlayerintheRoom** [comment]
> Hi, I've made a repository for restoring telegram html back up. Starting from html_to_txt of Suberbia. It works and it restore also the medias. The method is slow, though. If you want it to check it up and maybe change the code for your purpose it's here: https://github.com/lilium895/Telegram-chat-restorer-pyautogui-version

What happened to repository ? Why its gone :(

**Comment by 1mpalo** [comment]
> i think he means result.json file which is produced by telegram chat export on PC when you choose to export as JSON instead of HTML
> 
> btw, this script is not actually what are you looking for, it restores only text messaes, ignoring all content.
> You better try this: https://github.com/filippz/telegram_import
> It's really working and imports everything

Isn't this script able to import all the data, including photos, videos and other?
